src/main/webapp/app/exercises/text/assess-new/conflicts/text-feedback-conflicts.component.ts

```diff
@@ -0,0 +1,265 @@
+import { HttpErrorResponse, HttpResponse } from '@angular/common/http';
+import { JhiAlertService } from 'ng-jhipster';
+import { TranslateService } from '@ngx-translate/core';
+import { Location } from '@angular/common';
+
+import { TextAssessmentBaseComponent } from 'app/exercises/text/assess-new/text-assessment-base.component';
+import { TextSubmission } from 'app/entities/text-submission.model';
+import { TextAssessmentsService } from 'app/exercises/text/assess/text-assessments.service';
+import { TextBlockRef } from 'app/entities/text-block-ref.model';
+import { TextBlock, TextBlockType } from 'app/entities/text-block.model';
+import { TextExercise } from 'app/entities/text-exercise.model';
+import { Result } from 'app/entities/result.model';
+import { FeedbackConflict } from 'app/entities/feedback-conflict';
+import { AccountService } from 'app/core/auth/account.service';
+import { StructuredGradingCriterionService } from 'app/exercises/shared/structured-grading-criterion/structured-grading-criterion.service';
+
+import interact from 'interactjs';
+import * as moment from 'moment';
+
+@Component({
+    selector: 'jhi-text-feedback-conflicts',
+    templateUrl: './text-feedback-conflicts.component.html',
+    styleUrls: ['./text-feedback-conflicts.component.scss'],
+})
+export class TextFeedbackConflictsComponent extends TextAssessmentBaseComponent implements OnInit, AfterViewInit {
+    conflictingSubmissions?: TextSubmission[];
+    leftSubmission?: TextSubmission;
+    leftTextBlockRefs: TextBlockRef[];
+    leftUnusedTextBlockRefs: TextBlockRef[];
+    leftTotalScore: number;
+    leftFeedbackId: number;
+    rightSubmission?: TextSubmission;
+    rightTextBlockRefs: TextBlockRef[];
+    rightUnusedTextBlockRefs: TextBlockRef[];
+    rightTotalScore: number;
+    feedbackConflicts: FeedbackConflict[];
+    overrideBusy = false;
+    markBusy = false;
+    isOverrideDisabled = true;
+    isMarkingDisabled = true;
+    selectedRightFeedbackId?: number;
+
+    private get textBlocksWithFeedbackForLeftSubmission(): TextBlock[] {
+        return [...this.leftTextBlockRefs, ...this.leftUnusedTextBlockRefs]
+            .filter(({ block, feedback }) => block?.type === TextBlockType.AUTOMATIC || !!feedback)
+            .map(({ block }) => block!);
+    }
+
+    constructor(
+        private activatedRoute: ActivatedRoute,
+        private router: Router,
+        private location: Location,
+        protected accountService: AccountService,
+        protected assessmentsService: TextAssessmentsService,
+        protected jhiAlertService: JhiAlertService,
+        translateService: TranslateService,
+        protected structuredGradingCriterionService: StructuredGradingCriterionService,
+    ) {
+        super(jhiAlertService, accountService, assessmentsService, translateService, structuredGradingCriterionService);
+        const state = router.getCurrentNavigation()?.extras.state as { submission: TextSubmission };
```

Comment:
I think we discussed this before: If this is not defined, we should fetch this over the network so this page can be reloaded also.